tests/baselines/reference/abstractPropertyNegative.errors.txt

```diff
@@ -31,10 +31,12 @@ tests/cases/compiler/abstractPropertyNegative.ts(41,18): error TS2676: Accessors
         abstract m(): string;
         abstract get mismatch(): string;
                      ~~~~~~~~
-!!! error TS2380: 'get' and 'set' accessor must have the same type.
+!!! error TS2380: 'get' and 'set' accessor must have the same type 'number', but this 'get' accessor has the type 'string'.
+!!! related TS2730 tests/cases/compiler/abstractPropertyNegative.ts:10:18: The respective 'set' accessor has the type 'number'.
         abstract set mismatch(val: number); // error, not same type
                      ~~~~~~~~
-!!! error TS2380: 'get' and 'set' accessor must have the same type.
+!!! error TS2380: 'get' and 'set' accessor must have the same type 'string', but this 'get' accessor has the type 'number'.
+!!! related TS2730 tests/cases/compiler/abstractPropertyNegative.ts:11:18: The respective 'set' accessor has the type 'string'.

```

Comment:
this elaboration should be different, it should be about the `get` and not the `set` since this error is already about the `set` accessor.

Reply:
This is what I mentioned in my most recent comment above. As of now (with passing tests) the messages are shown twice and the second message is actually garbage. If I test only for the getter the output looks as desired (that's what I presume), but the tests fail. 
So what do we do?

Reply:
not sure i understand what you mean by "but the tests fail."

Reply:
The tests from `jake runtests-parallel` fail.
If I implement the function this way:
```
function checkAccessorDeclarationTypesIdentical(first: AccessorDeclaration, second:      
AccessorDeclaration, getAnnotatedType: (a: AccessorDeclaration) => Type | undefined, message: DiagnosticMessage) {
            const firstType = getAnnotatedType(first);
            const secondType = getAnnotatedType(second);
            if (firstType && secondType && !isTypeIdenticalTo(firstType, secondType)) {
                const typeNameSecondType = typeToString(secondType);

                if(isGetAccessor(first)){

                    const diagnostic: Diagnostic = error(first, message, typeNameSecondType);
                    addRelatedInfo(diagnostic, createDiagnosticForNode(first, Diagnostics.The_respective_set_accessor_has_the_type_0, typeNameSecondType));

                }
            }
        }
```
The output looks like this:
```
test_accessors.ts:3:9 - error TS2380: 'get' and 'set' accessor must have the same type 'string'.

3     get foo(): number { return 100; }
          ~~~

  test_accessors.ts:3:9
    3     get foo(): number { return 100; }
              ~~~
    The respective 'set' accessor has the type 'string'.
```
This is how I would expect the output to look like. But now the unit tests fail:
```
jake runtests-parallel
node lib/tsc.js -b src
node built/local/tsc.js -b built/local/typescriptServices.tsconfig.json
cp -r built/local/typescriptServices.js built/local/typescript.js
cp -r built/local/typescriptServices.js.map built/local/typescript.js.map
node lib/tsc.js -b src/testRunner
node built/local/run.js
Discovered 134 unittest suites.
Discovering runner-based tests...
Discovered 12550 test files in 672ms.
Starting to run tests using 4 threads...
Batching initial test lists...
Batched into 4 groups with approximate total file sizes of 2457413 bytes in each group. (99.6% of total tests batched)

  [▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․] 3 failing
[Worker 0] Unexpected error(s) found.  Error list is:

[Worker 0]   from: 1:8, to: 1:10, message: 'get' and 'set' accessor must have the same type 'number'.


[Worker 0]   from: 5:8, to: 5:9, message: Type '"30"' is not assignable to type 'number'.

Actual number of errors (2) does not match expected number (3)
  [▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬] ✖ 61200/61208 passing (5m)


```
What to make of this?

Reply:
The tests are baseline tests, so they will fail every time the output changes. and that is fine.
change the message, then run the tests (`jake runtests-parallel`), then accept the new baselines (`jake baseline-accept`).